client/pokemon/pokemon-full.view.html

```diff
@@ -343,15 +338,21 @@ <h3 style="margin-left:40px;">{{pokemon.parsedNickname}}</h3>
             <md-card class="summary" ng-show="pokemon.hasFullData">
               <md-card-header><span class="md-headline">Super Training</span></md-card-header>
               <md-card-content>
+                <div>
+                  <strong>Secret Training enabled? </strong>
```

Comment:
I wonder if it would be better to just display that muscles icon thing, since that's how people will recognize it.